frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/components/UploadStatus/style.ts

```diff
@@ -35,23 +36,12 @@ export const DatasetStatusTag = styled.div`
   align-self: flex-start;
   width: fit-content;
   padding: ${PT_GRID_SIZE_PX}px;
-  margin: 0 auto;
   display: flex;
   flex-direction: row;
   vertical-align: middle;
-  & > .${Classes.SPINNER}, .${Classes.ICON} {
```

Comment:
instead of styling by relying on HTML structure and hierarchy, which is brittle, we can use a styled component wrapper to style the `Spinner` component directly